packages/core/src/tree-view/tree.stories.mdx

```diff
@@ -0,0 +1,81 @@
+import { Meta, Props, Story, Preview, API } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Tree View" />
+
+# Tree View
```

Comment:
nit: add beta warning message like [this one](https://github.com/vmware/clarity/blame/next/packages/core/src/progress-circle/progress-circle.stories.mdx#L7)